src/properties/transform.rs

```diff
@@ -1457,23 +1457,25 @@ impl ToCss for Perspective {
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[cfg_attr(feature = "jsonschema", derive(schemars::JsonSchema))]
 #[cfg_attr(feature = "into_owned", derive(static_self::IntoOwned))]
-pub struct Translate {
-  /// The x translation.
-  pub x: LengthPercentage,
-  /// The y translation.
-  pub y: LengthPercentage,
-  /// The z translation.
-  pub z: Length,
+pub enum Translate {
+  /// The "none" keyword.
+  None,
```

Comment:
We could also use `Keyword(&'a str)` but not sure if it's worth it to do that honestly especially since keywords could all be known upfront _and_ the `'none'` value is the only one we care about right now.